src/main/java/org/opensearch/neuralsearch/processor/combination/RRFScoreCombinationTechnique.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.combination;
+
+import lombok.ToString;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.Map;
+
+@Log4j2
+/**
+ * Abstracts combination of scores based on reciprocal rank fusion algorithm
+ */
+@ToString(onlyExplicitlyIncluded = true)
+public class RRFScoreCombinationTechnique implements ScoreCombinationTechnique {
+    @ToString.Include
+    public static final String TECHNIQUE_NAME = "rrf";
+
+    // Not currently using weights for RRF, no need to modify or verify these params
+    public RRFScoreCombinationTechnique(final Map<String, Object> params, final ScoreCombinationUtil combinationUtil) {}
```

Comment:
I'm OK that weights are not supported in the first release. This class does nothing but adding all the scores together. I'm afraid it's too over designed to introduce a new class for such a single sum operation.

Reply:
I understand. For this PR, both params and combinationUtil are unused. You'd better delete them.

Reply:
ack, will do in next PR, please remind me if I forget